ddl/ddl_api.go

```diff
@@ -165,6 +186,7 @@ func (d *ddl) AlterSchema(ctx sessionctx.Context, stmt *ast.AlterDatabaseStmt) (
 		}
 	}
 
+	// TODO: Check if Placement Rules was set/changes? How to handle removal?
```

Comment:
For removal, you don't have to handle. You can't DROP a placement policy if it is in use (simplification we made earlier this week).